content/blog/2018-09-14-smapr.md

```diff
@@ -0,0 +1,637 @@
+---
+slug: "smapr"
+title: Mapping the 2018 East Africa floods from space with smapr
+package_version: 0.1.2
+authors:
+  - name: Max Joseph
+    url: https://github.com/mbjoseph
+date: 2018-09-14
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- data-access
+- geospatial
+---
+
+Hundreds of thousands of people in east Africa have been displaced and hundreds have died as a result of torrential rains which ended a drought but saturated soils and engorged rivers, resulting in extreme flooding in 2018.
+This post will explore these events using the R package smapr, which provides access to global satellite-derived soil moisture data collected by the NASA Soil Moisture Active-Passive (SMAP) mission and abstracts away some of the complexity associated with finding, acquiring, and working with the HDF5 files that contain the observations (shout out to [Laura DeCicco](https://github.com/ldecicco-USGS) and [Marco Sciaini](https://github.com/marcosci) for reviewing smapr, and [Noam Ross](https://github.com/noamross) for editing in the Ropensci onboarding process).
```

Comment:
can you link "NASA Soil Moisture Active-Passive (SMAP) mission" somewhere so readers can get more info if interested?

Reply:
Yep - done here: https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1